### PR TITLE
Add vuetify and plotly.js to wis2node-ui

### PR DESCRIPTION
**Pull request by webb-ben**
First commit re: https://github.com/wmo-im/wis2node/issues/29, adding [Vuetify](https://next.vuetifyjs.com/en/) and [plotly.js](https://plotly.com/javascript/) to wis2node-ui.

- Turn header and footer into vueitfy components (and all other components)
- Add WMO banner image to header and footer
- Add `/plot` route which provides a plot and UI to visualize (dummy) data. Spacing still needs some work.

Demo images:
image
image

### CODE REVIEW

**Comment by webb-ben** [comment]
NOTE:
`"@vue/cli-plugin-babel": "~4.5.0",` uses `"@vue/cli-service": "~4.5.0",` but `"vuetify": "^3.0.0-alpha.0",` requires `"@vue/cli-service": "~5.0.0-beta.3",` which sometimes results in a fail of `npm install`. The fix to this is to run `npm install --legacy-peer-deps`. This is what was run to generate the `package-lock.json` file in this PR.

[ Vuetify for Vue v3](https://next.vuetifyjs.com/en/introduction/roadmap/#in-development) is still under development with the beta version expected for December 2021. This is hopefully going to be resolved more cleanly in the future. The legacy dependencies also sometimes cause a `entrypoint size limit:` error when building the docker image. Running the build again without changing anything usually resolves this. 

Thinking of also adding github workflow to build and run wis2node-ui tests.

File: src/components/AppFooter.vue [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
 
 export default {
   name: "AppFooter",
+  template: "#app-footer",
   data() {
     return {
-      version: version,
-      name: name,
-      homepage: homepage,
+      version_: version,
```
Comment (tomkralidis) [inline src/components/AppFooter.vue:35]: Do the vars not work without the trailing `_` ?

File: src/components/HelloWorld.vue [hunk 2]
```diff
@@ -0,0 +1,129 @@
+<template>
```
Comment (tomkralidis) [inline src/components/HelloWorld.vue:1]: Do we need this file?

File: src/main.js [hunk 3]
```diff
@@ -1,6 +1,11 @@
+import "vuetify/styles";
 import { createApp } from "vue";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
+import vuetify from "./plugins/vuetify";
+import { loadFonts } from "./plugins/webfontloader";
```
Comment (tomkralidis) [inline src/main.js:7]: Should this be `import loadFonts from "./plugins/webfontloader";` ?

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: src/components/AppFooter.vue (hunk 1 above)
Comment (webb-ben) [reply src/components/AppFooter.vue:35]: Was getting some linting warning about exporting them as default but it does work

File: src/components/HelloWorld.vue (hunk 2 above)
Comment (webb-ben) [reply src/components/HelloWorld.vue:1]: no must've snuck in when i installed vuetify. oops

File: src/main.js (hunk 3 above)
Comment (webb-ben) [reply src/main.js:7]: I copy/pasted from vuetify docs but I can switch it and I am sure it'd work.

Comment (webb-ben) [reply src/main.js:7]: Nevermind loadFonts is not the default export from webfontloader. The build succeeds when I change this but the homepage doesn't render. Something to keep in mind going forward though?

**Pull request merged**